css.md

```diff
@@ -451,4 +452,122 @@ h6 {
 }
 ```
 
+## <a name='styled-components'>Styled Components</a>
+
+Not stricting speaking SASS, but some of the same ideas apply. Here are a few specific styled component considerations:
```

Comment:
`stricting` 🤔

Reply:
Yeah I guess we're not allowed to make up our own words